### PR TITLE
FAB-5168 - Fix default sort params for tasks list

### PR DESCRIPTION
**Pull request by pkandarpa-cs**
# Checklist:
Please check you fulfill ALL of the relevant checkboxes
- [ ] Notified docs of any potential USER-facing changes
- [x] Added short description of the change - with relevant motivation and context. 
- [x] Branch has the ticket number in its name (along with a ticket summary)
- [x] Commented the code, particularly in hard-to-understand areas
- [ ] Added tests that prove my fix is effective or that my feature works
- [x] Ran `npm test` and it passes
- [x] Changes generate no new warnings
- [x] Changes are backward compatible with older clusters

### CODE REVIEW

**Review by yreddy-CS — APPROVED** [review]

File: bin/cortex-tasks.js [hunk 1]
```diff
@@ -56,7 +58,7 @@ program
     .option('--filter <filter>', 'A Mongo style filter to use.')
     .option('--limit <limit>', 'Limit number of records', DEFAULT_LIST_LIMIT_COUNT)
     .option('--skip <skip>', 'Skip number of records', DEFAULT_LIST_SKIP_COUNT)
-    .option('--sort <asc|desc|sort>', 'Sort asc|desc on startTime or Mongo style sort statement.', 'desc')
+    .option('--sort <asc|desc|sort>', 'Sort asc|desc on startTime or Mongo style sort statement.', GET_DEFAULT_SORT_CLI_OPTION(DEFAULT_LIST_SORT_PARAMS.startTime))
```
Comment (yreddy-CS) [inline bin/cortex-tasks.js:61]: TO be consistent with the other list help, maybe desc can be "A Mongo style sort statement to use in the query."

**Review by yreddy-CS — APPROVED** [review]

File: bin/cortex-tasks.js (hunk 1 above)
Comment (pkandarpa-cs) [reply bin/cortex-tasks.js:61]: done

**Comment by pkandarpa-cs** [comment]
PR build seems to fail because of an artifactory login issue

File: bin/cortex-tasks.js [hunk 2]
```diff
@@ -56,7 +58,7 @@ program
     .option('--filter <filter>', 'A Mongo style filter to use.')
     .option('--limit <limit>', 'Limit number of records', DEFAULT_LIST_LIMIT_COUNT)
     .option('--skip <skip>', 'Skip number of records', DEFAULT_LIST_SKIP_COUNT)
-    .option('--sort <asc|desc|sort>', 'Sort asc|desc on startTime or Mongo style sort statement.', 'desc')
+    .option('--sort <mongo-style-sort-expr>', 'Mongo style sort statement.', GET_DEFAULT_SORT_CLI_OPTION(DEFAULT_LIST_SORT_PARAMS.startTime))
```
Comment (jgielstra-cs) [inline bin/cortex-tasks.js:61]: asc|desc are still valid ?

File: package.json [hunk 3]
```diff
@@ -117,6 +117,7 @@
     ]
   },
   "overrides": {
-    "got": "11.8.5"
+    "got": "11.8.5",
+    "json5": "^2.2.2"
```
Comment (jgielstra-cs) [inline package.json:121]: I just do `npm audit fix` and patch the package-lock.json. ... Seems like less risk of forgotten maintenance..?

File: src/constants.js [hunk 4]
```diff
@@ -64,6 +64,10 @@ module.exports.ALLOWED_QUERY_FIELDS = {
         filter: ['status', 'start', 'end', 'skillName', 'agentName'],
         sort: ['status', 'start', 'end', 'skillName', 'agentName'],
     },
+    TASK: {
```
Comment (jgielstra-cs) [inline src/constants.js:67]: Eww lots of dual maintenance :(

**Review by jgielstra-cs — APPROVED** [review]
Just a few comments...

File: bin/cortex-tasks.js (hunk 2 above)
Comment (pkandarpa-cs) [reply bin/cortex-tasks.js:61]: fixed

**Pull request merged**